bootstrapvz/plugins/puppet/tasks.py

```diff
@@ -79,14 +152,4 @@ def run(cls, info):
         os.remove(manifest_dst)
 
         hosts_path = os.path.join(info.root, 'etc/hosts')
-        sed_i(hosts_path, '127.0.0.1\s*{hostname}\n?'.format(hostname=hostname), '')
-
-
-class EnableAgent(Task):
```

Comment:
Could you keep the `EnableAgent` task at the bottom? Moving it makes it hard to read the diff.